src/context/EarnManagerProvider/components/Deposit/Deposit.tsx

```diff
@@ -83,13 +83,17 @@ enum InputType {
 enum Field {
   FiatAmount = 'fiatAmount',
   CryptoAmount = 'cryptoAmount',
-  Slippage = 'slippage'
+  Slippage = 'slippage',
+  TxStatus = 'txStatus',
+  UsedGasFee = 'usedGasFee'
```

Comment:
Are these being used by the form? If not, these fields shouldnt be defined here? We should lift them to the parent

Reply:
Ah, that makes sense. They are not used in the form, I've hoisted them.